src/core_modules/capture-core/components/Pages/Enrollment/EnrollmentPage.actions.js

```diff
@@ -2,30 +2,39 @@
 import { actionCreator } from '../../../actions/actions.utils';
 
 export const enrollmentPageActionTypes = {
-    ENROLLMENT_PAGE_INFORMATION_FETCH: 'StartFetchingEnrollmentPageInformation',
-    ENROLLMENT_PAGE_INFORMATION_LOADING_FETCH: 'LoadingOnFetchingEnrollmentPageInformation',
-    ENROLLMENT_PAGE_INFORMATION_ERROR_FETCH: 'ErrorOnFetchingEnrollmentPageInformation',
-    ENROLLMENT_PAGE_INFORMATION_SUCCESS_FETCH: 'SuccessOnFetchingEnrollmentPageInformation',
-
-    ENROLLMENT_PAGE_OPEN: 'OpenEnrollmentPage',
-    ENROLLMENT_PAGE_CLEAN: 'CleanEnrollmentPage',
+    INFORMATION_FETCH: 'EnrollmentPage.Fetch',
+    INFORMATION_USING_TEI_ID_FETCH: 'EnrollmentPage.StartFetchingUsingTeiId',
+    INFORMATION_USING_ENROLLMENT_ID_FETCH: 'EnrollmentPage.StartFetchingUsingEnrollmentId',
+    INFORMATION_LOADING_FETCH: 'EnrollmentPage.LoadingOnFetching',
+    INFORMATION_ERROR_FETCH: 'EnrollmentPage.ErrorOnFetching',
+    INFORMATION_SUCCESS_FETCH: 'EnrollmentPage.SuccessOnFetching',
+
+    PAGE_OPEN: 'EnrollmentPage.Open',
+    PAGE_CLEAN: 'EnrollmentPage.CleanOnUnmount',
+    DEFAULT_VIEW: 'EnrollmentPage.DefaultView',
```

Comment:
Renamed those a little. My logic is now that we are exporting as const the  action types aka `enrollmentPageActionTypes` we dont have to repeat in every action the `ENROLLMENT_PAGE`, because usually it will be that we use these actions in a way similar to this `enrollmentPageActionTypes.INFORMATION_FETCH`. This covers already where the action is coming from. 

Before we had the `enrollmentPageActionTypes.ENROLLMENT_PAGE_INFORMATION_FETCH`. Now that way we repeat ourselves. IMO